anchore_engine/db/entities/common.py

```diff
@@ -51,12 +51,12 @@ class should override this function with an impl that serializes those types pro
 
         :return: string
         """
+
+        # TODO: should normalize DateTime's to use RFC3339 in the future to make the timezone explicit
         return dict(
             (
                 key,
-                value
-                if type(value) != datetime.datetime
-                else datetime_to_rfc3339(value),
+                value if type(value) != datetime.datetime else value.isoformat(),
```

Comment:
@zhill, timestamp values in the api response will not contain UTC timezone - 'Z' after this change. Is that the intended purpose of this change?

Reply:
This is the original form in master. The change you're seeing is an intermediate commit. Changing it to RFC3339 broke some other cases because it parses with a populated timezone, so things like comparison with datetime.datetime.utcnow() will raise exceptions. I wanted to make the change to standardize on RFC3339 but reverted it due to blast radius of that comparison issue.

Reply:
Aah, gotcha. The comment makes sense now